pkg/scheduling/volumeusage.go

```diff
@@ -187,6 +203,19 @@ func (v *VolumeUsage) validate(ctx context.Context, kubeClient client.Client, po
 	return podPVCs, nil
 }
 
+func (v *VolumeUsage) discoverDefaultStorageClassName(ctx context.Context, kubeClient client.Client) (*string, error) {
+	storageClassList := &storagev1.StorageClassList{}
+	if err := kubeClient.List(ctx, storageClassList); err != nil {
+		return nil, err
+	}
+	if sc, ok := lo.Find(storageClassList.Items, func(sc storagev1.StorageClass) bool {
```

Comment:
Apparently you can have more than one annotated as the default and we need to pick the newest one, see https://github.com/kubernetes/kubernetes/pull/110559